anime/Services/ImportScheduleService.php

```diff
@@ -232,16 +232,22 @@ private function loadMapping() : array {
 
         for ($i = 1 /* skip the header */; $i < count($mappingLines); ++$i) {
             $mappingLine = str_getcsv(trim($mappingLines[$i]));
-            if (count($mappingLine) < 6)
+            if ((count($mappingLine) < 6) && !((count($mappingLine) == 2) && ($mappingLine[1] != '-')))
```

Comment:
Hmm, right.
I was thinking of the case of generating a new configuration from something else.
However, I do think the changes to the destination_team_program argument might be useful, as it makes the code a lot easier to understand.